cwms_radar_api/src/test/java/fixtures/TomcatServer.java

```diff
@@ -121,6 +119,11 @@ public Realm getRealm(){
      */
     public void start() throws LifecycleException {
         tomcatInstance.start();
+        try {
```

Comment:
why is the sleep needed?  Can you add a comment on what happens if the sleep isn't there or how to see if 5 seconds isn't long enough?

Reply:
actually I don't think it is. something was going on that was confusing me but if I remember right it was some other obvious thing like I had a file name slightly wrong so something wasn't loading.